source/includes/_openstack.md

```diff
@@ -0,0 +1,3 @@
+# OpenStack Plugin
+
+The CloudMC OpenStack plugin incorporates Nova and Neutron features into CloudMC's API. It also enforces CloudMC's environment-centric multi-tenancy model and RBAC over top of OpenStack.
```

Comment:
We will not be limited to Nova and Neutron. Maybe "The CloudMC OpenStack plugin incorporates various OpenStack services such as Nova and Neutron into..."